CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 # Change Log
 All notable changes to this project will be documented in this file.
-## [unreleased]
+## [17.2.2]
```

Comment:
The "-2017" is just an artifact of us having two separate releases for each IJ product.
I wouldn't like to make it too public..